x-pack/plugins/security_solution/common/api/search_strategy/model/filter_query.ts

```diff
@@ -80,4 +80,4 @@ const esQuerySchema = z.union([
   jsonObjectSchema,
 ]);
 
-export const filterQuery = z.union([z.string(), z.undefined(), esQuerySchema]);
+export const filterQuery = z.union([z.string(), z.undefined(), z.any()]).optional();
```

Comment:
aren't `z.undefined()` and `.optional()` doing the same thing here?

Reply:
yup, thanks